src/service/BookingService.java

```diff
@@ -0,0 +1,37 @@
+package service;
+
+import java.math.BigDecimal;
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+import database.dao.BookingDataDAO;
+import database.dto.BookingDataDTO;
+import database.dto.PaymentMethodDTO;
+
+public class BookingService {
+	
+	private final BookingDataDAO bookingDataDAO;
+	private final BigDecimal pricePerNight=new BigDecimal("5000");
+	private BigDecimal finalPrice = null;
```

Comment:
This variable should be declared within the method, since it use is independent per caller, remember variables here are shared across the instance, so far there will be only one instance of BookingService, so it will be shared by everyone using it.